mediator/med.F90

```diff
@@ -835,14 +851,14 @@ subroutine InitializeIPDv03p1(gcomp, importState, exportState, clock, rc)
 
     if (mastertask) then
        write(logunit,*)
-       write(logunit,*) "atm_present="//trim(atm_present)
-       write(logunit,*) "lnd_present="//trim(lnd_present)
-       write(logunit,*) "ocn_present="//trim(ocn_present)
-       write(logunit,*) "ice_present="//trim(ice_present)
-       write(logunit,*) "rof_present="//trim(rof_present)
-       write(logunit,*) "wav_present="//trim(wav_present)
-       write(logunit,*) "glc_present="//trim(glc_present)
-       write(logunit,*) "med_present="//trim(med_present)
+       write(logunit,*) "atm_present = "//trim(atm_present)//" - atm_name = "//trim(atm_name)
```

Comment:
What is atm_name if atm_present is false?   maybe instead of this you could have
if(atm_present) write(logunit,*) "atm_name = "//trim(atm_name)

Reply:
Strings are initialized to blank values. Is this okay? See line [81](https://github.com/ESCOMP/CMEPS/pull/154/files/45eb748ee19cfed10575a2ce2c7ac72d618db43c#diff-fd9e77eeb56b02d6d6a3b93a1e5d47bb1d6f83b3557a22ba1c17dca9a0d35ee4R81) in mediator/esmFlds.F90.

Reply:
Option 2: We can also set the atm_name to something like "NA" if you don't like " " here:
[https://github.com/hafs-community/CMEPS/blob/support/HAFS/mediator/med.F90#L778](https://github.com/hafs-community/CMEPS/blob/support/HAFS/mediator/med.F90#L778)

If we do add if (atm_present) the code won't be as concise, the code for each component will look like...
```
if (atm_present) then
   write(logunit,*) "atm_present = "//trim(atm_present)//" - atm_name = "//trim(atm_name)
else
   write(logunit,*) "atm_present = "//trim(atm_present)
endif
```

Reply:
why write anything at all if it's false? 
if (atm_present) write(logunit,*) "atm_name = "//trim(atm_name)

is all you need.

Reply:
med.F90 currently writes whether a component is present or not.
https://github.com/ESCOMP/CMEPS/blob/master/mediator/med.F90#L836

Reply:
it's not worth the fight